src/Extensions/Abstractions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary> Extension methods for the <see cref="IServiceCollection"/> class</summary>
+	public static class ServiceCollectionExtensions
+	{
+		/// <summary>Add IServiceContext to ServiceCollection</summary>
```

Comment:
>summary [](start = 7, length = 7)

Add `<param>` and `<returns>` throughout this class

Reply:
The method was deleted, sorry